taxi/admin.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+
+from taxi.models import Driver, Car, Manufacturer
+
+
+@admin.register(Driver)
+class DriverAdmin(UserAdmin):
+    list_display = UserAdmin.list_display + ('license_number',)
+    fieldsets = UserAdmin.fieldsets + (("Additional",
+                                        {"fields": ("license_number", )}),)
+

```

Comment:
Add licence_number to the add_fieldset so you can fill this field while adding a driver.